uwsgi_tasks/tasks.py

```diff
@@ -230,6 +238,14 @@ def create(cls, **kwargs):
 class RuntimeTask(BaseTask):
     executor = TaskExecutor.RUNTIME
 
+    def execute_now(self):
+        try:
+            _serialize(self.args)
+            _serialize(self.kwargs)
+        except AttributeError as ex:
+            raise RuntimeError("Task arguments are not pickle-able") from ex
+        super().execute_now()
```

Comment:
This package still support Python 2, so this line will generate `TypeError`, please use tox to confirm that your code pass the tests.